### DISCUSSION TITLE
Edit/delete bookmark?

### DISCUSSION THREAD

**Discussion by hejops** [plateaukao/einkbro#181] [General]
As far as I can tell, there is no way to edit or delete bookmarks.

**Comment by plateaukao** [comment]
@hejops 
Please long click on the bookmark you want to edit or delete.

**Comment by XetaPompei88** [comment]
The advice doesn't help me. I can delete pages from history tab by long click, but i can't delete bookmark.

**Reply by plateaukao** [reply]
Not from history tab. Please delete them from bookmark list, which can be triggered by clicking on bookmark button. <img width="40" alt="image" src="https://github.com/user-attachments/assets/7f88c4e4-81fb-4ab2-a4f3-6b79f7ea8b15" />
